tox.ini

```diff
@@ -12,9 +12,10 @@ norecursedirs = .*
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
-    django110: Django>=1.10,<1.11
     django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2
```

Comment:
This should have an upper bound in preparation for when the next major version of Django is released.